apps/costcontrol/js/costcontrol.js

```diff
@@ -588,6 +588,10 @@ var CostControl = (function() {
         var manifestURL = request.result.appManifestURL;
         // ignore the 'null' manifestURL
         if (manifestURL && manifestURL !== 'null') {
+          if (request.result.browsingTrafficOnly &&
```

Comment:
Could you add an intermmediate variable with the test? Something like `isBrowser`?